draft-ietf-rtcweb-jsep.xml

```diff
@@ -455,12 +455,21 @@ setRemote(ANSWER) |                                   |
         <section title="RtpSenders" anchor="sec.rtpsenders">
 
           <t>RtpSenders allow the application to control how RTP media
-          is sent.</t>
+          is sent. An RtpSender is conceptually responsible for the
+          outgoing RTP stream(s) described by an m= section. This includes
+          encoding the attached MediaStreamTrack, sending RTP media
+          packets, sending RTCP sender reports, and processing received
```

Comment:
One issue here is that sender reports also include information about received media. That is, a SR is a superset of an RR, and thereby must include information about **received** media. Also, when SRs are sent, RRs are typically not sent.

My suggestion would be to abstract this text somewhat to refer to "generating and processing RTCP about the outgoing RTP stream(s)", and the text below to "generating and processing RTCP about the incoming RTP stream(s)".